deploy/docs/v2-18-migration.md

```diff
@@ -0,0 +1,24 @@
+# Upgrading from v2.17 to v2.18
+
+## Upgrade OpenTelemetry Operator CRDs
+
+### Do I need to do this?
+
+You need to do this if you have the OpenTelemetry Operator subchart enabled with `opentelemetry-operator.enabled: true` in your values file.
+
+### What do I need to do?
+
+Run the following commands on the cluster before upgrading the chart:
+
+```shell
+kubectl apply --server-side -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator-0.24.0/charts/opentelemetry-operator/crds/crd-opentelemetrycollector.yaml
```

Comment:
do we need the `--force-conflicts` option like here:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/ba08b58707e58cdb50da0032f0be1e4c8997f0dc/docs/v3-migration-doc.md#upgrade-kube-prometheus-stack

Reply:
oh nvm, I see that we have `-f`

Reply:
It's a good point that the flag should be probably in long format, I'll change that :+1:

Reply:
Added the flag `--force-conflicts`.